src/Dependencies/Collections/README.md

```diff
@@ -0,0 +1,11 @@
+﻿# Microsoft.CodeAnalysis.Collections
+
+This package contains shared code implementing specialized collection types.

```

Comment:
If this is going to be shared to other repositories, please doc it explicitly.

Reply:
Updated notes, and filed #49991 to address a remaining gap in those notes.